runtime/doc/lua.txt

```diff
@@ -2280,56 +2281,78 @@ find({names}, {opts})                                          *vim.fs.find()*
     searches are recursive and may search through many directories! If {stop}
     is non-nil, then the search stops when the directory given in {stop} is
     reached. The search terminates when {limit} (default 1) matches are found.
-    The search can be narrowed to find only files or or only directories by
+    The search can be narrowed to find only files or only directories by
     specifying {type} to be "file" or "directory", respectively.
+    
+     Examples: >
+     
+       vim.fs.find("aFileOrDirectoryNextToNeovim")
+       => "C:/Users/jdoe/aFileOrDirectoryNextToNeovim"
+       
+       vim.fs.find("allFilesOrDirectoriesAboveNeovim", {upward = true, limit = math.huge})
+       => "/home/jdoe/allFilesOrDirectoriesAboveNeovim"
+       => "/home/allFilesOrDirectoriesAboveNeovim"
+       
+       vim.fs.find("aFileInHomeDirectory", {path = "/home"})
+       => "/home/jdoe/archive/aFileInHomeDirectory"
+<
 
     Parameters: ~
-      • {names}  (string|table|fun(name: string): boolean) Names of the files
+    
+      • {names}  (string or table or fun(name: string): boolean) Names of the files
                  and directories to find. Must be base names, paths and globs
-                 are not supported. If a function it is called per file and
-                 dir within the traversed directories to test if they match.
+                 are not supported. The function is called per file and
+                 directory within the traversed directories to test if they match {names}.
+		 
       • {opts}   (table) Optional keyword arguments:
                  • path (string): Path to begin searching from. If omitted,
-                   the current working directory is used.
+                   the current |current-directory| is used.
```

Comment:
This doesn't match the docstring in the source code. Make sure you regenerate the docs with `./scripts/gen_vimdoc.py`.

Reply:
How do run `./scripts/gen_vimdoc.py`?

Reply:
Exactly like this.